pygmt/datasets/earth_relief.py

```diff
@@ -3,12 +3,13 @@
 The grids are available in various resolutions.
 """
 import xarray as xr
+from packaging.version import Version
 
-from .. import which
+from .. import clib, which
 from ..exceptions import GMTInvalidInput
 
 
-def load_earth_relief(resolution="01d"):
+def load_earth_relief(resolution="01d", pixel_reg=None):
```

Comment:
I personally prefer an argument like `registration="pixel"` or `registration="gridline"`, just like **-r** option in most GMT modules.

Reply:
I agree that `registration="pixel"` is much better.

Reply:
Yep, this can be arranged.